src/isar/state_machine/state_machine.py

```diff
@@ -125,8 +125,8 @@ def __init__(
                 {
                     "trigger": "pause",
                     "source": [self.initiate_state, self.monitor_state],
-                    "dest": self.stop_state,
-                    "before": self._pause,
+                    "dest": self.paused_state,
```

Comment:
This might be a breaking change for step_wise robots

Reply:
We can add a new transition and if it's not stepwise mission call that new transition that doesn't go trough the stop state. Will add a TODO

Reply:
![image](https://github.com/equinor/isar/assets/108514046/88b4ccb9-22f7-4bb4-9da7-1fa9fc425ee7)


![image](https://github.com/equinor/isar/assets/108514046/705327fd-00ad-4e89-8562-a6872e263901)